app/models/devise_token_auth/concerns/active_record_support.rb

```diff
@@ -2,7 +2,7 @@ module DeviseTokenAuth::Concerns::ActiveRecordSupport
   extend ActiveSupport::Concern
 
   included do
-    if Rails::VERSION::MAJOR >= 7 && Rails::VERSION::MINOR >= 1
+    if Gem::Version.new(Rails::VERSION::STRING) >= "7.1"
```

Comment:
This fails in ruby < 3.1 with `ArgumentError: comparison of Gem::Version with String failed` . Maybe try with `Rails.gem_version >= Gem::Version.new("7.1.x")` ?

Reply:
Interesting. There I pushed your suggestion with a minor change to account for pre-release versions

<img width="560" alt="image" src="https://github.com/lynndylanhurley/devise_token_auth/assets/6373536/f3f26f34-0210-4fbe-b560-95f8bc21cece">

Reply:
Interesting. There I pushed your suggestion with a minor change to account for pre-release versions

<img width="569" alt="image" src="https://github.com/lynndylanhurley/devise_token_auth/assets/6373536/8bcf3dae-7801-4334-aaaf-40fac4583140">